tools/plugins/com.liferay.ide.project.ui/plugin.xml

```diff
@@ -66,6 +66,25 @@
                class="org.eclipse.core.resources.IResource">
          </selection>
       </wizard>
+      <wizard
+            category="com.liferay.ide.newWizards"
+            finalPerspective="com.liferay.ide.eclipse.ui.perspective.liferay"
+            icon="icons/e16/liferay_logo_16.png"
+            id="com.liferay.ide.project.ui.workspace.NewLiferayWorkspaceWizard"
+            name="Liferay Workspace"
+            preferredPerspectives="com.liferay.ide.eclipse.ui.perspective.liferay,org.eclipse.jst.j2ee.J2EEPerspective"
+            project="true">
+         <class class="com.liferay.ide.project.ui.workspace.NewLiferayWorkspaceWizard">
+			<parameter name="liferay_project" value="true"/>
+			<parameter name="menuIndex" value="12" />
+         </class>
+         <description>
+            Create a Liferay Workspace for developing Liferay Plugins
```

Comment:
This should say "Create a new Liferay Workspace project for developing modules and themes for Liferay 7"